roles/slurm_management/templates/configure_slurm_accounting_db.bash

```diff
@@ -111,20 +114,42 @@ sacctmgr -i create qos set Name='priority-medium'
 sacctmgr -i modify qos Name='priority-medium' set \
     Description='priority-medium' \
     Priority=20 \
+    Preempt='leftover-short,leftover-medium,leftover-long' \
     UsageFactor=2 \
     GrpSubmit=2500  MaxSubmitJobsPU=500   MaxWall=1-00:00:00 \
-    GrpTRES=cpu={{ [1, (cluster_cores_total | float * 0.5) | int] | max }},mem={{ [1000, (cluster_mem_total | float * 0.5) | int] | max }} \
+    GrpTRES=cpu={{ [1, (cluster_cores_total | float * 0.6) | int] | max }},mem={{ [1000, (cluster_mem_total | float * 0.6) | int] | max }} \
     MaxTRESPU=cpu={{ [1, (cluster_cores_total | float * 0.2) | int] | max }},mem={{ [1000, (cluster_mem_total | float * 0.2) | int] | max }}
 
 sacctmgr -i create qos set Name='priority-long'
 sacctmgr -i modify qos Name='priority-long' set \
     Description='priority-long' \
     Priority=20 \
+    Preempt='leftover-short,leftover-medium,leftover-long' \
     UsageFactor=2 \
     GrpSubmit=250   MaxSubmitJobsPU=50   MaxWall=7-00:00:00 \
-    GrpTRES=cpu={{ [1, (cluster_cores_total | float * 0.2) | int] | max }},mem={{ [1000, (cluster_mem_total | float * 0.2) | int] | max }} \
+    GrpTRES=cpu={{ [1, (cluster_cores_total | float * 0.3) | int] | max }},mem={{ [1000, (cluster_mem_total | float * 0.3) | int] | max }} \
     MaxTRESPU=cpu={{ [1, (cluster_cores_total | float * 0.1) | int] | max }},mem={{ [1000, (cluster_mem_total | float * 0.1) | int] | max }}
 
+#
+# QoS interactive
+#
+sacctmgr -i create qos set Name='interactive'
+sacctmgr -i modify qos Name='interactive' set \
+    Description='Highest priority Quality of Service level for interactive sessions.' \
+    Priority=30 \
+    UsageFactor=1 \
```

Comment:
I see that the preemptive option is set in the _interactive-short_, but not in the _interactive_. Is this later specified or missing?

Reply:
That is correct. It was incorrectly specified for the ```priority``` QoS. The "main" QoS levels are what users can request / submit to, but no job will ever run in these main/routing QoS levels. Jobs will get a sub QoS assigned based on requested walltime, so it is the `priority-short`, `priority-medium` and `priority-long` sub QoS levels that need things like `preempt` set. For `interactive` there is only `interactive-short`; no `-medium` nor `-long` sub QoS here.